backend/hitas/services/condition_of_sale.py

```diff
@@ -54,6 +54,11 @@ def create_conditions_of_sale(owners: list["Owner"]) -> list[ConditionOfSale]:
                 if ownership.id == other_ownership.id:
                     continue
 
+                # Don't create conditions of sale between two ownerships to the same apartment.
+                # They are in the same sale but for different owners.
+                if other_ownership.apartment == apartment:
+                    continue
+
```

Comment:
For directly related things such as this PR and #335 (Both even modify the same file & function) feel free to just include all of the changes in the same PR, or base the latter PR on top of the first one. This way there is no need to resolve conflicts when one of them is merged

Reply:
Ok, I'll try to remember. The other PR is now merged and conflicts are solved.